src/test/libcephfs/acl.cc

```diff
@@ -307,3 +307,52 @@ TEST(ACL, Disabled) {
   free(acl_buf);
   ceph_shutdown(cmount);
 }
+
+TEST(ACL, SnapdirACL) {
```

Comment:
can we check that the chmod on the directory, also changes the ACLs of its snapdir? similar to 'Chmod' test above.

Reply:
I missed seeing the test SnapdirAttrs.